extensions/wikia/AdEngine/js/SevenOneMediaHelper.js

```diff
@@ -227,7 +227,7 @@ define('ext.wikia.adEngine.sevenOneMediaHelper', [
 	function generateSoiKeywords() {
 		log('generateSoiKeywords', 'debug', logGroup);
 
-		var i, len, param, val, valIndex, valLen, keywords = [];
+		var i, len, param, val, valIndex, valLen, keywords = [], kruxSegments;
```

Comment:
I don't understand this comment of a merged changes :japanese_ogre: Is it about variables declaration in one line? I think it's `// also acceptable`.